keras_rs/src/layers/embedding/distributed_embedding_test.py

```diff
@@ -318,6 +320,32 @@ def test_model_fit(self, input_type, use_weights):
         with self._strategy.scope():
             layer = distributed_embedding.DistributedEmbedding(feature_configs)
 
+        def _create_keras_input(
+            feature_config: config.FeatureConfig, dtype: types.DType
+        ) -> keras.KerasTensor:
+            # JAX doesn't allow us to directly mark an input as ragged, since
+            # Keras+JAX doesn't directly support ragged tensors.
+            # To work around this, explicitly modify the attribute after creation.
+            tensor_input = keras.layers.Input(
+                feature_config.input_shape[1:],
+                dtype=dtype,
+                sparse=(input_type == "sparse"),
+                ragged=False,
+            )
+            tensor_input._ragged = input_type == "ragged"
```

Comment:
How about this as a work-around?

This let's us keep Keras as-is (not actually supporting ragged tensors), but we can still "detect" a ragged symbolic tensor to avoid the warning from the functional model.

Reply:
Yeah, I suppose that's ok too. Up to you.

Reply:
Alright, marking as `sparse`.  PTAL.